cmake/BLTMacros.cmake

```diff
@@ -270,7 +270,10 @@ macro(blt_add_target_link_flags)
             # In CMake 3.13+, LINK_FLAGS was converted to LINK_OPTIONS.
             # This now supports generator expressions but expects a list
             # not a string
-            set_property(TARGET ${arg_TO} APPEND PROPERTY LINK_OPTIONS ${arg_FLAGS})
+            string(REPLACE " " ";" arg_FLAGS_LIST ${arg_FLAGS})
```

Comment:
Please rename arg_FLAG_LIST and arg_FLAG to _flag_list and _flag respective to not make it look like it is coming in as an argument.

Reply:
done